lighthouse-treemap/app/src/main.js

```diff
@@ -121,7 +122,10 @@ class TreemapViewer {
     }
 
     for (const [group, depthOneNodes] of Object.entries(this.depthOneNodesByGroup)) {
-      makeOption({type: 'group', value: group}, `All ${group}`);
+      const allLabel = {
```

Comment:
what's the purpose of this object? are there about to be many types here?

Reply:
maybe, i don't know, I'm being careful here.